constellations/gemini-admin/src/user/userAFH.js

```diff
@@ -0,0 +1,187 @@
+                    });
+                    this.fetchAFHs(afhIds);
+                })
+            )
+            .catch((err) => alert("Could not fetch user: " + err));
+    };
+
+    fetchAFHs = (afhIds) => {
+        let searchArray = new Set(afhIds);
+        API.get("api/askforhelp/all")
+            .then((res) => {
+                var userAFHs = [];
+                var afhs = [];
+                res.data.forEach((afh) => {
+                    if (searchArray.has(afh.id)) {
+                        userAFHs.push(afh);
+                    } else {
+                        afhs.push(afh);
+                    }
+                });
+
+                userAFHs = _.sortBy(userAFHs, ["date"]);
+                afhs = _.sortBy(afhs, ["date"]);
+                this.setState({
+                    userAFHs: userAFHs,
+                    afhs: afhs,
+                });
+            })
+            .catch((err) => alert("Could not fetch afhs: " + err));
+    };
+
+    onAFHChange = (e) => {
+        this.setState({
+            afhId: e.target.value,
+        });
+    };
+
+    onClickSchedule = () => {
+        const userAFH = {
+            userId: parseInt(this.state.id),
+            afhId: parseInt(this.state.afhId),
+        };
+
+        API.post("api/userafhs/create", userAFH)
+            .then(() => {
+                this.fetchUser();
+            })
+            .catch((err) => alert("Could not schedule AFH: " + err));
+    };
+
+    render = () => {
+        const user = this.state.user;
+
+        var fullName = user.firstName;
+        if (user.middleName) {
+            fullName += " " + user.middleName + " " + user.lastName;
+        } else {
+            fullName += " " + user.lastName;
+        }
+
+        const rows = this.state.userAFHs.map((afh, index) => {
+            const status = moment().isBefore(afh.date)
+                ? "Will Attend"
+                : "Attended";
+            return (
+                <div className="row" key={index}>
+                    <span className="column">
+                        {moment(afh.date).format("l")}
+                    </span>
+                    <span className="large-column">{afh.title}</span>
+                    <span className="column">{afh.subject}</span>
+                    <span className="column status">{status}</span>
+                </div>
+            );
+        });
+
+        const afhOptions = this.state.afhs.map((afh, index) => {
+            return (
+                <option key={index} value={afh.id}>
+                    {moment(afh.date).format("l") +
+                        " " +
+                        afh.subject +
+                        " " +
+                        afh.timeString}
+                </option>
+            );
+        });
+
+        var schedule = (
+            <span>
+                There are no AFH sessions to choose from. Please add one{" "}
+                <Link to="/afh/add">here</Link>
+            </span>
```

Comment:
I think if you just do this, it should be easier:
`<span>...</span><Link>...</Link>`